ChangeLog.rst

```diff
@@ -9,6 +9,12 @@
 * Fix #109: Fix for unexpanded &lt; &gt; &amp;
 * Fix #143: Fix line wrapping for the lines starting with bold
 
+2017.7.27
+=========
+----
+
+* Adds support for numeric bold text indication in `font-weight`,
```

Comment:
You don't have to make a new release section such as "2017.7.27".
You need to the changelog to future release section which is "0000.00.00".